.vscode/tasks.json

```diff
@@ -98,7 +98,7 @@
     },
     {
         "label": "generate compiler code",
-        "command": "${workspaceFolder}/.dotnet/dotnet",
```

Comment:
`.dotnet` directory doesn't have to exist locally. So, running the task fails. This fixed it for me.

Reply:
I'm not sure what the implications of this change are, and it doesn't seem directly related to the generator change. Also, there are other uses of the same `"command"` in this file.

Let's avoid changing this file in this PR.